CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 ### Added
 
+- [#2252](https://github.com/thanos-io/thanos/pull/2252) Query: add new `--store.strict-mode` flag. More information available [here](/docs/proposals/202001_thanos_query_health_handling.md).
```

Comment:
Looks like this was not updated. (:

Reply:
Oops, updated. Thanks for the quick review!